py-rse-index.Rmd

```diff
@@ -72,100 +125,46 @@ Sami Virtanen
 
 ### Prerequisites
 
-Learners must be able to:
+Learners must already be able to:
 
--   Write reusable functions to analyze tabular data using [R][r] or [Python][python].
+-   Write functions to analyze tabular data using [Python][python].
+    (A companion volume covers this material using [R][r].)
 -   Keep track of their work using [Git][git] and [GitHub][github].
```

Comment:
I don't think we should assume they know anything about git and github. It would only take a minor update to our git at the command line chapter to make it accessible for people who haven't used git before. (Because it doesn't differ that much from the Software Carpentry git at the command line lesson, which doesn't assume git knowledge)

Reply:
I think we'd have to expand our Git coverage significantly if we want to make it accessible to people who've never used it before. I've changed this to be "saving their work in a Git repository using	a graphical interface"

Reply:
I teach git at the command line in my [PyAOS Data Carpentry lessons](https://carpentrieslab.github.io/python-aos-lesson/05-git/index.html) and I don't assume that people have used a git graphical interface. The only real difference between that PyAOS lesson and what's in our "Git at the command line" chapter is a short introduction to what version control is and an explanation of the staging area. I'd be happy to submit a PR to add those elements to our git at the command line chapter ASAP so that we can remove the prerequisite about needing to have used graphical git. I feel pretty strongly about not excluding people from reading our book if they haven't used git before - lots of people who would benefit from our book will not have used git before.

Reply:
:+1: please go ahead - I've removed the dependency on Git from this chapter.